pkg/sanity/node.go

```diff
@@ -241,6 +241,21 @@ var _ = DescribeSanity("Node Service", func(sc *TestContext) {
 			Expect(ok).To(BeTrue())
 			Expect(serverError.Code()).To(Equal(codes.InvalidArgument))
 		})
+
+		It("should fail when the volume is missing", func() {
+
+			_, err := c.NodeUnpublishVolume(
+				context.Background(),
+				&csi.NodeUnpublishVolumeRequest{
+					VolumeId:   sc.Config.IDGen.GenerateUniqueValidVolumeID(),
+					TargetPath: sc.StagingPath,
+				})
+			Expect(err).To(HaveOccurred())
```

Comment:
`Expect.To` without some additional description leads to failure messages that cannot be understood without locating the source code. I know, most of the existing assertions are like that, but for new ones let's use `Expect.To(..., "some description")`.